concepticondata/conceptlists/Maciejewski-2016-100.tsv

```diff
@@ -0,0 +1,201 @@
+Maciejewski-2016-100-56	56	hide	2	the skin of one of the larger animals such as a buffalo or cow	LEATHER OR HIDE	3326	23.92	0.66	73	26.39	1.42	2.93	6	6	1.83	4.3	4.51	5.15
+Maciejewski-2016-100-57	57	hiding	1	the act or condition of concealing or being concealed	HIDE (CONCEAL)	602	74.38	0.65	97	14.58	1.16	2.95	3	1	1.78	NA	NA	NA
+Maciejewski-2016-100-58	58	hiding	2	a thrashing, flogging, or severe beating			25.37	0.65	97	14.58	1.16	2.95	3	1	1.78	NA	NA	NA
+Maciejewski-2016-100-59	59	host	1	a person who provides hospitality such as food, entertainment, or lodging for guests	HOST	1545	81.69	0.76	75	34.33	1.54	3.4	6	11	1.96	NA	NA	NA
+Maciejewski-2016-100-60	60	host	2	a very large number of people or things			17.99	0.76	75	34.33	1.54	3.4	6	11	1.96	NA	NA	NA
+Maciejewski-2016-100-61	61	jam	1	to force or pack tightly into a small space			38.01	0.39	47	9.82	0.99	1.66	13	11	1.6	5.69	5.63	5.29
+Maciejewski-2016-100-62	62	jam	2	a sweet spread usually made by cooking crushed fruit and sugar			58.71	0.39	47	9.82	0.99	1.66	13	11	1.6	5.69	5.63	5.29
+Maciejewski-2016-100-63	63	keen	1	extremely sharp; able to cut readily and finely			44.09	0.78	90	38.02	1.58	3.5	5	6	2.08	2.3	3.36	5.62
+Maciejewski-2016-100-64	64	keen	2	a loud, mournful wailing for the dead			7.74	0.78	90	38.02	1.58	3.5	5	6	2.08	2.3	3.36	5.62
+Maciejewski-2016-100-65	65	launch	1	to propel with force			87.06	0.83	81	34.02	1.53	2.68	8	8	1.86	4.02	NA	4.63
+Maciejewski-2016-100-66	66	launch	2	a large open boat propelled by a motor			12.94	0.83	81	34.02	1.53	2.68	8	8	1.86	4.02	NA	4.63
+Maciejewski-2016-100-67	67	lawn	1	a stretch or plot of ground planted with grass or other low-growing ground covers and usually mowed short			90.35	0.92	97	10.59	1.02	2.86	2	1	1.42	5.98	5.88	5.01
+Maciejewski-2016-100-68	68	lawn	2	a fine, sheer, woven fabric, usually of cotton or linen			9.65	0.92	97	10.59	1.02	2.86	2	1	1.42	5.98	5.88	5.01
+Maciejewski-2016-100-69	69	lean	1	to bend or incline			59.18	0.39	56	13.41	1.13	3.37	11	10	1.73	3.39	3.95	4.91
+Maciejewski-2016-100-70	70	lean	2	having little flesh	LEAN (MEAT)	2219	40.62	0.39	56	13.41	1.13	3.37	11	10	1.73	3.39	3.95	4.91
+Maciejewski-2016-100-71	71	limp	1	to walk in an uneven, labored, lame manner	LIMP	366	53.88	0.32	52	5.62	0.75	3.05	5	5	1.59	4.18	4.81	4.5
+Maciejewski-2016-100-72	72	limp	2	not stiff, rigid, or firm; floppy or flabby			46.12	0.32	52	5.62	0.75	3.05	5	5	1.59	4.18	4.81	4.5
+Maciejewski-2016-100-73	73	lock	1	a mechanical device for preventing entry through a door or window or into a safe, usually opened with a key or combination	LOCK	1596	81.86	0.75	77	24.19	1.38	2.95	12	15	1.67	5.32	5.65	5.88
+Maciejewski-2016-100-74	74	lock	2	a curl or portion of hair			18.14	0.75	77	24.19	1.38	2.95	12	15	1.67	5.32	5.65	5.88
+Maciejewski-2016-100-75	75	mail	1	the system organized to send and deliver letters, parcels, and the like; postal system	MAIL	167	87.68	0.85	83	36.05	1.56	3.22	6	7	1.7	5.65	5.08	5.94
```

Comment:
I would leave it unmapped because MAIL on concepticon refers to physical objects (letters and parcels) rather than to a postal system.